training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/submit_smp_trainer.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<blockquote>You can also specify different training parameters here such as <code>batch size</code>, <code>tensor parallelism</code>, <code>data parallelism</code>, and <code>fp16</code> which will affect if your model can fit on your instance configuration.</blockquote><p>List the exact parameters for <code>batch size</code>, <code>tensor parallelism</code>, <code>data parallelism</code>. Otherwise, do not use the code-in-line style.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3577/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Ok, will update!